events/voiceStateUpdate.js

```diff
@@ -123,26 +128,28 @@ module.exports = {
 			await player.handler.locale('MUSIC_ALONE_RESUMED');
 			return;
 		}
-		// user has nothing to do with us
+		// User from other channel that has nothing to do with us
 		if (oldState.channelId !== player?.channelId) return;
-		// user didn't leave the vc
+		// User didn't leave the channel, but their voice state changes
 		if (newState.channelId === oldState.channelId) return;
-		// vc still has people
+		/** Statements for when a user LEAVEs */
+		// Channel still has humans
 		if (oldState.channel.members.filter(m => !m.user.bot).size >= 1) return;
-		// 24/7 mode enabled, ignore
+		// Avoid pauseTimeout if 24/7 is enabled
 		if (await data.guild.get(guild.id, 'settings.stay.enabled')) return;
-		// nothing is playing so we just leave
+		// Nothing is playing so we just leave
 		if (!player.queue.current || !player.playing && !player.paused) {
 			logger.info({ message: `[G ${player.guildId}] Disconnecting (alone)`, label: 'Quaver' });
 			player.handler.locale('MUSIC_ALONE');
 			await player.handler.disconnect();
 			return;
 		}
-		// rare case where the bot sets pause timeout after setting timeout
-		// another weird issue where pause timeout is set after stage ends
+		// Rare case where the bot sets pauseTimeout after setting timeout
+		// Another weird issue where pauseTimeout is set after stage ends
```

Comment:
// Ensure that the bot does not set pauseTimeout if timeout already exists
// Ensure that the bot does not set pauseTimeout after a stage ends